cdci_data_analysis/analysis/job_manager.py

```diff
@@ -95,7 +95,7 @@ def __init__(self,
 
     @staticmethod
     def get_allowed_job_status_values():
-        return ['done', 'main_done', 'failed', 'progress', 'submitted', 'ready', 'unknown', 'unaccessible','aliased', 'post-processing']
```

Comment:
Why removed back? Will this keep triggering the 410 error?

Reply:
Please consider the rest of the code. All unexpected callbacks are now treated as progress updates, since it's better to send it as so for the user.